src/test/java/com/microsoft/jfr/RecordingConfigurationTest.java

```diff
@@ -43,13 +50,43 @@ public void jfcFileFromInputStreamCanBeRead() {
         executeRecording("sampleJfcFile.jfc");
     }
 
+    @Test
+    public void mapConfiguration() {
+
+        Map<String, String> recordingConfigAsMap = new HashMap<>();
+        recordingConfigAsMap.put("jdk.ObjectAllocationInNewTLAB#enabled", "true");
+        recordingConfigAsMap.put("jdk.ObjectAllocationOutsideTLAB#enabled", "true");
+
+        RecordingConfiguration recordingConfiguration = new RecordingConfiguration.MapConfiguration(recordingConfigAsMap);
+
+        IItemCollection recordingContent = excecuteRecordingWithConfig(recordingConfiguration);
```

Comment:
missing a null check

Reply:
Good point. I added [this](https://github.com/jeanbisutti/jfr-streaming/blob/c372f6ee1796d40f63976bee23f535e149967a5d/src/test/java/com/microsoft/jfr/RecordingConfigurationTest.java#L104). I looked at `JfrLoaderToolkit.loadEvents(File file)` code, it never seems to return null.